src/components/Form/DateRange/DateRange.jsx

```diff
@@ -290,6 +293,7 @@ DateRange.defaultProps = {
   minDateEqualTo: false,
   maxDate: new Date(),
   maxDateEqualTo: false,
+  showPresent: true,
```

Comment:
An annoying naming comment:  I think `allowPresent` might be a little more semantic....just a personal preference though

Reply:
Yeah sort of blindly following the existing convention of using `show`, but no reason not to slowly start making those more semantic. Updated https://github.com/18F/e-QIP-prototype/pull/935/commits/c259479af0f37ae5f651b09c1dc307a11ddf61b1